frontend/src/components/personal/achievements/Achievements.test.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react'
+import { render, screen } from '@testing-library/react'
+import { Provider } from 'react-redux'
+import '@testing-library/jest-dom'
+
+import {store} from '../../../index'
+import {AchievementsBoard} from './Achievements'
+
+describe('NotificationForm', () => {
+  it('title in place', () => {
```

Comment:
Does Achievements.tsx has only title in it? How does it work and what functional does it have? Can we check that popover is shown on hover?